src/metabase/email/messages.clj

```diff
@@ -226,3 +226,121 @@
                        (vec (cons :div (for [result results]
                                          (render/render-pulse-section timezone result)))))]
     (render-message-body "metabase/email/pulse" (pulse-context body pulse) (seq @images))))
+
+(defn pulse->alert-condition-kwd
+  "Given an `ALERT` return a keyword representing what kind of goal needs to be met."
+  [{:keys [alert_above_goal alert_condition card creator] :as alert}]
+  (if (= "goal" alert_condition)
+    (if (true? alert_above_goal)
+      :meets
+      :below)
+    :rows))
+
+(defn- first-card
+  "Alerts only have a single card, so the alerts API accepts a `:card` key, while pulses have `:cards`. Depending on
+  whether the data comes from the alert API or pulse tasks, the card could be under `:card` or `:cards`"
+  [alert]
+  (or (:card alert)
+      (first (:cards alert))))
+
+(defn- default-alert-context
+  ([alert]
+   (default-alert-context alert nil))
+  ([alert alert-condition-map]
+   (let [{card-id :id, card-name :name} (first-card alert)]
+     (merge {:questionURL (url/card-url card-id)
+             :questionName card-name
+             :emailType    "alert"
+             :sectionStyle render/section-style
+             :colorGrey4   render/color-gray-4
+             :logoFooter   true}
+            (random-quote-context)
```

Comment:
are we including the random quotes about data in the alert emails?

Reply:
Yes, all of the emails use [the same footer](https://github.com/metabase/metabase/blob/master/src/metabase/email/_footer.mustache)